### DISCUSSION TITLE
cannot approve my educational discount request

### DISCUSSION THREAD

**Discussion by Xue-Yue** [community/community#49522] [GitHub Education]
I uploaded my transcripts and proof of school (including time), admission notice. 
But none of these will get me through the application!!!

**Comment by samus-aran** [comment]
Hi @Xue-Yue 👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
